examples/framework/preferences/minimalEditor/README.md

```diff
@@ -0,0 +1,9 @@
+
+## Minimal Preferences Editor ##
+
+This folder provides an example of a minimal but fully functional preferences editor. It has:
+- only one preference
+- no enactor
+- no localization (Note that this is not best practice!)
+
+This code is the basis for the Preferences Framework tutorial.
```

Comment:
You have the tutorial hosted on your personal github page right now, and because it's going into the infusion-docs we could probably figure out what the url would be. Or we could commit the tutorial first and then you could add the link in before committing this. Any preference?